Document managing secrets with Cloud UI (#3000)
influxdata/docs-v2: content/influxdb/cloud/security/secrets/add.md

---
title: Add secrets
description: Add secrets using the {{< cloud-name "short" >}} UI, `influx` CLI, or the InfluxDB API.
influxdb/cloud/tags: [secrets, security]
menu:
  influxdb_cloud:
    parent: Manage secrets
weight: 301
---

Add secrets using the {{< cloud-name "short" >}} UI, `influx` command line interface (CLI) or the InfluxDB API.

- [Add a secret using the InfluxDB Cloud UI](#add-a-secret-using-the-influxdb-cloud-ui)
- [Add a secret using the influx CLI](#add-a-secret-using-the-influx-cli)
- [Add a secret using the InfluxDB API](#add-a-secret-using-the-influxdb-api)

## Add a secret using the InfluxDB Cloud UI

1. In the {{< cloud-name "short" >}} UI, click **Settings**.

    {{< nav-icon "settings" >}}
2. Click the **Secrets** tab.
3. Click **Add Secret**.
4. Enter a *key* and a *value*.
   {{% warn %}}

> If will need it again, be sure to safely store your secret value.

Reviewer: ```suggestion
To access the secret again, be sure to safely store your secret value.
```
